### DISCUSSION TITLE
Unsigned Installers

### DISCUSSION THREAD

**Discussion by trevorboydsmith** [opentoonz/opentoonz#6475] [Q&A]
the most recent version 1.0.2 installer is not signed and when you install under windows 10 you get a popup complaining about "unknown" source.

if possible it might be good to add "signing" of the code as an additional step for your build on windows 10.

(personally this is low priority. I personally do not have issues clicking "ok" when presented with a dialog from windows 10 complaining about unsigned code but other users may.)

if signing the code is too bothersome somehow, then maybe providing an md5sum hash that users can use to check against.

**Comment by blurymind** [comment]
Another point to bring out is the license information which is all in
japanese for the installer. So all the users not speaking the language are
asked to agree to a license they cant read in order to install the
application

**Comment by trevorboydsmith** [comment]
@blurymind "license information which is all in japanese for the installer" is a completely separate issue. if there isn't already an issue for "license information which is all in japanese for the installer" then feel free to create a new issue.

**Comment by RodneyBaker** [comment]
This issue likely needs to be renamed or closed.
The status of 'unsigned' installers is still unresolved but v1.02 has long ago been superseded.

Also:  The subtopical issues related to english license has been resolved.

**Comment by ghost** [comment]
I think it's still relevant, so I just changed the title.

**Comment by RodneyBaker** [comment]
Thanks Jeremy

**Comment by RodneyBaker** [comment]
Is a signed license something that just needs to be funded?  As in purchasing a certificate or certificate granting autorization?  If that is the case we mostly just need to know how much it would cost to consider whether to move forward.

**Comment by RodneyBaker** [comment]
I started a review of this process but haven't gotten far.
As such this addition will likely slip to 2019 development cycle.

Of note, many users see the notification of unsigned certificate as a potential threat.
If you see someone struggling with this please assist and let them know that github itself has effective safeguards in place and the signed signatures are still pending.

**Comment by artisteacher** [comment]
MacOS has the same issue. This has to turn off some users over fears of security risks. It also makes it a little bit more of a hassle to install.

On the mac side, to release a signed app, you need to be part of the Apple Developer Program. There's a $99 yearly fee. [The fee can be waived in some cases](https://developer.apple.com/support/membership-fee-waiver/). It would certainly be worth it to apply for the waiver in case we qualify.

**Comment by RodneyBaker** [comment]
Converting this to a discussion as there are currently no plans to sign the installer.
This doesn't rule that out but given 10 years of having the installers not signed it seems reasonable that direction will be continued.
